db/queries/categories.js

```diff
@@ -0,0 +1,74 @@
+const db = require('../connection');
+
+const getAllCategory = () => {
+  const query = `SELECT * FROM categories;`;
```

Comment:
We would need to fetch the current user's categories, not all categories, so we do need a `where` clause.